pandas-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/README.md

```diff
@@ -0,0 +1,56 @@
+# Pipeline
+
+> *Note:* This is a `README.md` boilerplate generated using `Kedro {{ cookiecutter.kedro_version }}`.
+
+## Overview
+
+This pipeline:
+1. splits the data into training dataset and testing dataset using a configurable ratio found in `conf/base/parameters.yml`
+2. runs a simple 1-nearest neighbour model (`make_prediction` node) and makes prediction dataset.
+3. reports the model accuracy on a test set (`report_accuracy` node)
+
+## Pipeline inputs
+
```

Comment:
The only input here is actually `example_iris_data` and `parameters`. Please would you add `example_iris_data` with description "Example iris data containing columns _<names of columns>_" and move the `parameters` bit here? Then all the below datasets let's call "Pipeline intermediate outputs". And under `Pipeline outputs` say "None".